src/MCEventsProcessing/MCEventsProcessing.jl

```diff
@@ -153,3 +153,4 @@ function _generate_waveform( drift_paths::Vector{<:EHDriftPath{T}}, charges::Vec
 end
 
 
+function run_geant4_simulation end
```

Comment:
Are you planning on adding a docstring in this PR or will that happen separately?